indexing-service/src/test/java/org/apache/druid/indexing/seekablestream/SeekableStreamSupervisorSpecTest.java

```diff
@@ -131,8 +135,21 @@ public void setUp()
     monitorSchedulerConfig = EasyMock.mock(DruidMonitorSchedulerConfig.class);
     supervisorStateManagerConfig = EasyMock.mock(SupervisorStateManagerConfig.class);
     supervisor4 = EasyMock.mock(SeekableStreamSupervisor.class);
+    lagStats = EasyMock.mock(LagStats.class);
+    timesLagStatsMaxCalled = 0;
+    timesLagStatsSumCalled = 0;
 
     EasyMock.expect(spec.getContextValue(DruidMetrics.TAGS)).andReturn(null).anyTimes();
+    EasyMock.expect(lagStats.getAggregateForScaling()).andReturn(AggregateFunction.SUM).anyTimes();
+    EasyMock.expect(lagStats.getMetric(AggregateFunction.SUM)).andAnswer(() -> {
+      timesLagStatsSumCalled++;
+      return 0L;
+    }).anyTimes();
+    EasyMock.expect(lagStats.getMetric(AggregateFunction.MAX)).andAnswer(() -> {
+      timesLagStatsMaxCalled++;
+      return 0L;
+    }).anyTimes();
+    EasyMock.replay(lagStats);
```

Comment:
These are weird expectations. Why do we care how many times sum and max were called?